Blockchain/readme.md

```diff
@@ -44,6 +44,11 @@
   - [2)Infura](https://infura.io/)
   - [3)Quicknode](https://www.quicknode.com/)
 
+## Layer 2 Solutions
+- [Polygon](https://wiki.polygon.technology/)
+- [Lightning Network](http://lightning.network/docs/)
+
```

Comment:
Remove the empty spaces here, They seem to be unnecessary and redundant

Reply:
Done